test/test_ccgs.py

```diff
@@ -65,37 +65,37 @@ def test_primal_obj(offset):
     assert res.success
     lbfgsb_gap = res.x.reshape((nx, ny))
 
-    np.testing.assert_allclose(ccg_with_restart_gap, ccg_without_restart_gap, atol=1e-3)
-    np.testing.assert_allclose(ccg_with_restart_gap, lbfgsb_gap, atol=1e-3)
-    np.testing.assert_allclose(lbfgsb_gap, ccg_without_restart_gap, atol=1e-3)
+    np.testing.assert_allclose(CCGWithRestart_gap, CCGWithoutRestart_gap, atol=1e-3)
+    np.testing.assert_allclose(CCGWithRestart_gap, lbfgsb_gap, atol=1e-3)
+    np.testing.assert_allclose(lbfgsb_gap, CCGWithoutRestart_gap, atol=1e-3)
 
     # ##########TEST MEAN VALUES#######################################
     mean_val = np.mean(lbfgsb_gap)
     # ####################POLONSKY-KEER##############################
-    res = ccg_with_restart.constrained_conjugate_gradients(
+    res = CCGWithRestart.constrained_conjugate_gradients(
         system.primal_objective(offset, gradient=True),
         system.primal_hessian_product,
         init_gap,
         gtol=gtol,
         mean_value=mean_val)
 
     assert res.success
-    ccg_with_restart_gap_mean_cons = res.x.reshape((nx, ny))
+    CCGWithRestart_gap_mean_cons = res.x.reshape((nx, ny))
 
     # ####################BUGNICOURT###################################
-    ccg_without_restart.constrained_conjugate_gradients(system.primal_objective
-                                                        (offset, gradient=True),
-                                                        system.
-                                                        primal_hessian_product,
-                                                        x0=init_gap,
-                                                        mean_val=mean_val,
-                                                        gtol=gtol
-                                                        )
+    CCGWithoutRestart.constrained_conjugate_gradients(system.primal_objective
+                                                      (offset, gradient=True),
+                                                      system.
+                                                      primal_hessian_product,
+                                                      x0=init_gap,
+                                                      mean_val=mean_val,
+                                                      gtol=gtol
+                                                      )
     assert res.success
 
-    ccg_without_restart_gap_mean_cons = res.x.reshape((nx, ny))
+    CCGWithoutRestart_gap_mean_cons = res.x.reshape((nx, ny))
```

Comment:
Why did you change these? Codestyle is pep-8, which is lowercase snake_case for variables. There are other instances where this has been changed.

I believe this was a replace all because the module name changed in NuMPI

Reply:
yes exacrly. Sorry for that